scripts/generate-google3-docs/resource-reference/generated/resource-docs/gkehub/gkehubfeaturemembership.md

```diff
@@ -1102,6 +1102,7 @@ spec:
   projectRef:
     name: gkehubfeaturemembership-dep-acm
   location: global
```

Comment:
Can we use the same location for `GKEHubFeatureMembership` and `GKEHubMembership`?

Reply:
Ditto, see the above comment https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/2525#discussion_r1722151189